docs/docusaurus/versioned_docs/version-1.6.0/orc8r/RDS_upgrade.md

```diff
@@ -0,0 +1,149 @@
+* Logs from an RDS upgrade (9.6.22 -> 9.6.23 -> 12.8) have been captured [_here_](https://gist.github.com/sudhikan/8e42985bc0db13512c9cd602d8acab3a) for reference.
+* Between each upgrade step, please verify that you are able to login to the NMS, access the subscriber page, modify values, and gateways are successfully checking-in. 
+* It is also recommended to take snapshots before each upgrade. Refer to the link in the prerequisites section for more information.
+
+## Cleanup 
+
+* Modify main.tf as follows:
+
+`orc8r_db_apply_immediately`` = ``"false"`
+
+* Run `terraform plan` and `terraform apply`. These are not expected to have any impact other than to disable immediate major upgrades of the RDS instance.
+
+```
+An execution plan has been generated and is shown below.
+Resource actions are indicated with the following symbols:
+  ~ update in-place
+
+Terraform will perform the following actions:
+
+  # module.orc8r.aws_db_instance.default will be updated in-place
+  ~ resource "aws_db_instance" "default" {
+      ~ apply_immediately                     = true -> false
+        id                                    = "orc8rdb"
+        name                                  = "orc8r"
+        tags                                  = {}
+        # (48 unchanged attributes hidden)
+    }
+
+Plan: 0 to add, 1 to change, 0 to destroy.
+```
+
+* After terraform has finished running, run `terraform show` to validate the DB settings.
+
+```
+# module.orc8r.aws_db_instance.default:
+resource "aws_db_instance" "default" {
+    address                               = "orc8rdb.abcdy9gi9jk5o.eu-west-2.rds.amazonaws.com"
+    allocated_storage                     = 64
+    allow_major_version_upgrade           = true
+    apply_immediately                     = false
+    arn                                   = "arn:aws:rds:eu-west-2:************:db:orc8rdb"
+    auto_minor_version_upgrade            = true
+    availability_zone                     = "eu-west-2a"
+    backup_retention_period               = 7
+    backup_window                         = "01:00-01:30"
+    ca_cert_identifier                    = "rds-ca-2019"
+    copy_tags_to_snapshot                 = false
+    customer_owned_ip_enabled             = false
+    db_subnet_group_name                  = "orc8r"
+    delete_automated_backups              = true
+    deletion_protection                   = false
+    enabled_cloudwatch_logs_exports       = []
+    endpoint                              = "orc8rdb.abcdy9gi9jk5o.eu-west-2.rds.amazonaws.com:5432"
+    engine                                = "postgres"
+    engine_version                        = "12.8"
+    engine_version_actual                 = "12.8"
+```
+
+> **_NOTE:_** If running terraform init --upgrade again in the future and the Magma version is 1.6 (Glacier Peak) or older, the `orc8r_db_apply_immediately = "false"` needs to be removed from the main.tf else Terraform will raise errors.
```

Comment:
What's this line mean?

Reply:
Okay I made a mistake here... @jaredmullane we can remove this. 

I was thinking that if the underlying variables didn't exist then it would error out. I think if the underlying variables in `db.tf` and `variables.tf` do not exist, nothing will happen as terraform will ignore this value.